sdk/nodejs/cmd/pulumi-language-nodejs/testdata/projects/l1-empty/package.json

```diff
@@ -5,6 +5,6 @@
 	},
 	"dependencies": {
 		"typescript": "^4.0.0",
-		"@pulumi/pulumi": "../../artifacts/pulumi-pulumi-1.0.1.tgz"
+		"@pulumi/pulumi": "../../artifacts/pulumi-pulumi-3.111.2.tgz"
```

Comment:
This is another sadness of this system. We end up having to regen all the snapshots whenever we bump the version.

Reply:
Yeah that's really awkward, I would really like to not have to upgrade snapshots every time we update the version.  Especially for patch releases we want to get out quickly that's another step that can go wrong.

I wonder if we can do something different here, e.g. maybe have a comparison function that excludes the appropriate files from being compared, or that knows how to compare them semantically?  This could be per language e.g. passed in as callback to `RunLanguageTests` or something?